odoo/addons/base/ir/ir_attachment.py

```diff
@@ -94,13 +94,14 @@ def _get_path(self, bin_data, sha):
     def _file_read(self, fname, bin_size=False):
         full_path = self._full_path(fname)
         r = ''
-        try:
-            if bin_size:
-                r = human_size(os.path.getsize(full_path))
-            else:
-                r = base64.b64encode(open(full_path,'rb').read())
-        except (IOError, OSError):
-            _logger.info("_read_file reading %s", full_path, exc_info=True)
+        if os.path.exists(full_path):
```

Comment:
This will return true for folders - is this the desired behaviour?
ref: https://docs.python.org/3.5/library/os.path.html#os.path.exists

Reply:
Good point!

It would have the equivalent  behavior as the original function in that case.

Its been a while since I looked at this (probably means it wasn't as big of a problem as I thought originally) I'll have to look if rest of the call chain expects the `IsADirectoryError`which would be thrown. 

Thanks for the heads up :)